Выравнивание

Kelit opened the pull request:
1) Добавил проверку на заголовок, если это заголовок применяем выравнивание к нему, а если нет, то к тексту параграфа
2) Изменил в Standarts  GetAlignment на то чтобы он возвращал JustificationValues
3) Аналогично в Gost1 и GostTest

Commit:
Выравнивание

1) Добавил проверку на заголовок, если это заголовок применяем выравнивание к нему, а если нет, то к тексту параграфа
2) Изменил в Standarts  GetAlignment на то чтобы он возвращал JustificationValues
3) Аналогично в Gost1 и GostTest

SapiPapik commented on StandardsLibrary/GOST1.cs:
```diff
@@ -39,6 +41,21 @@ public Gost1(string path)
+                    return JustificationValues.Both;
+            }
+        }
+
```
Вот эта штука уже реализована у Вас в данном [PR](https://github.com/parshikovalexey/mwalb/pull/8/files)
Смотри `OpenXmlHelperLibrary/Extension.cs`

Commit:
Убрал пробелы

Commit:
Merge branch 'dev' into alignment

Commit:
Fix

1) Убрал  JustificationValues
2) В ProcessingOpneXml реализовал Extension.GetJustificationByString

goffman committed:
Merge branch 'dev' into alignment

# Conflicts:
#	ProcessDocumentCore/Processing/ProcessingOpenXml.cs
